Formula/bash-preexec.rb

```diff
@@ -1,5 +1,5 @@
 class BashPreexec < Formula
-  desc "preexec and precmd functions for Bash just like Zsh."
```

Comment:
I know there are none now, but in the interests of consistency (so that none pop up again), I suppose we probably should. https://github.com/Homebrew/brew/pull/2795